docs/Getting Started.md

```diff
@@ -0,0 +1,59 @@
+# Getting Started
+
+## Clone Serai
+```
+git clone https://github.com/serai-dex/serai.git
+```
+
+## Build and Run Serai
+
+### Install Rust
+
+#### Linux
+```
+curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
+```
+
+#### Windows
+
+Use WSL2 and Linux instructions.
+
+
+#### Mac
+
+```
+brew install rustup
+```
+
+### Install Solidity Compiler
+```
+sudo pip3 install solc-select
+solc-select install 0.8.16
+solc-select use 0.8.16
+```
+### Install Other Dependencies

```

Comment:
These are Debian/Ubuntu specific other-dependencies (if actually required, as discussed in another issue).